cloud_governance/cloud_resource_orchestration/clouds/aws/ec2/monitor_cro_instances.py

```diff
@@ -77,7 +76,7 @@ def __monitor_instances(self):
         for ticket_id, cluster_data in cluster_tickets.items():
             for cluster_id, cluster_values in cluster_data.items():
                 cluster_values['cluster_name'] = cluster_id.split('/')[-1]
-                monitored_ticket_ids[ticket_id].append(cluster_values)
+                monitored_ticket_ids.setdefault(ticket_id, []).append(cluster_values)
```

Comment:
Is that ok, default is empty list ?
Why not to set it before the loop ?

Reply:
yes, we have two types of resources cluster and non-cluster. This loop is for cluster data. If we didn't find the non-cluster data default is empty. So we need to set the default as an empty list.

Reply:
ok